### PR TITLE
study import canUserAccessPhi

### PR DESCRIPTION
**Pull request by labkey-matthewb**
#### Rationale
Fix for import dataset with PHI columns and PHI not enabled.

#### Related Pull Requests
Same fix as for lists in https://github.com/LabKey/platform/pull/5689

#### Changes
-

### CODE REVIEW

File: core/src/org/labkey/core/admin/AdminController.java [hunk 1]
```diff
@@ -1626,11 +1628,12 @@ public void setFileName(String fileName)
     @RequiresPermission(AdminPermission.class)
     public class ViewValidationResultsAction extends SimpleViewAction<ViewValidationResultsForm>
     {
-        @Override
```
Comment (labkey-adam) [inline core/src/org/labkey/core/admin/AdminController.java:1629]: Why is `@Override` being removed?

File: core/src/org/labkey/core/admin/AdminController.java [hunk 2]
```diff
@@ -4970,7 +4973,11 @@ protected HttpView<?> getTabView(ExportFolderForm form, boolean reshow, BindExce
         {
             form.setExportType(PageFlowUtil.filter(getViewContext().getActionURL().getParameter("exportType")));
 
-            form.setExportPhiLevel(ComplianceService.get().getMaxAllowedPhi(getContainer(), getUser()));
+            ComplianceFolderSettings settings = ComplianceService.get().getFolderSettings(getContainer(), User.getAdminServiceUser());
+            PhiColumnBehavior columnBehavior = null==settings ? PhiColumnBehavior.show : settings.getPhiColumnBehavior();
+            PHI maxAllowedPhiForExport = PhiColumnBehavior.show == columnBehavior ? PHI.Restricted : ComplianceService.get().getMaxAllowedPhi(getContainer(), getUser());
+            form.setExportPhiLevel(maxAllowedPhiForExport);
+
```
Comment (labkey-adam) [inline core/src/org/labkey/core/admin/AdminController.java:4981]: Should we push all this into a different method on ComplianceService? Maybe something for the future.

**Review by labkey-adam — APPROVED** [review]

File: core/src/org/labkey/core/admin/AdminController.java (hunk 1 above)
Comment (labkey-matthewb) [reply core/src/org/labkey/core/admin/AdminController.java:1629]: oops?

File: core/src/org/labkey/core/admin/AdminController.java (hunk 2 above)
Comment (labkey-matthewb) [reply core/src/org/labkey/core/admin/AdminController.java:4981]: yes, after the dust settles on the spec...

**Pull request merged**